static/src/javascripts/lib/robust.js

```diff
@@ -19,23 +19,24 @@ const catchErrors = (fn: Function): ?Error => {
     return error;
 };
 
-const log = (module: string, error: Error): void => {
+const logError = (module: string, error: Error): void => {
     if (window.console && window.console.warn) {
         window.console.warn('Caught error.', error.stack);
     }
 
     reportError(error, { module }, false);
 };
 
-const catchErrorsAndLog = (name: string, fn: Function): void => {
+const catchAndLogError = (name: string, fn: Function): void => {
     const error = catchErrors(fn);
 
     if (error) {
-        log(name, error);
+        logError(name, error);
     }
 };
 
-const context = (modules: Array<any>): void =>
-    modules.forEach(([name, fn]) => catchErrorsAndLog(name, fn));
+const catchErrorsWithContext = (modules: Array<any>): void =>
```

Comment:
@gustavpursche and I were so happy with the name `context`, [mirroring Raven's API](https://docs.sentry.io/clients/javascript/usage/#context-wrap) 😭 

Do you think it's not descriptive enough?